apps/backend/src/config/redis.ts

```diff
@@ -4,11 +4,33 @@ import { envVars } from './env';
 
 const logger = createDefaultLogger();
 
+// TODO: Debug - Log minimal à retirer (sans le mot de passe)
+logger.info(
+  {
+    REDIS_HOST: envVars.REDIS_HOST ?? '(absent)',
+    REDIS_PORT: envVars.REDIS_PORT ?? '(absent)',
+    REDIS_USERNAME: envVars.REDIS_USERNAME ?? '(absent)',
+    REDIS_PASSWORD: envVars.REDIS_PASSWORD ? '(present)' : '(absent)',
```

Comment:
```suggestion
    REDIS_PASSWORD: envVars.REDIS_PASSWORD ? `(present, ${envVars.REDIS_PASSWORD.length} chars)` : '(absent)',
```

Je ferais bien ça, juste pour avoir un contrôle minimal